workflow/signal/signal.go

```diff
@@ -49,6 +52,9 @@ func ExecPodContainerAndGetOutput(ctx context.Context, restConfig *rest.Config,
 	if err != nil {
 		return err
 	}
+	var executorSignalTimeout = env.LookupEnvDurationOr("EXECUTOR_SIGNAL_TIMEOUT", 2*time.Minute)
```

Comment:
nit: I'd write this as `executorSignalTimeout := env.LookupEnvDurationOr("EXECUTOR_SIGNAL_TIMEOUT", 2*time.Minute)`, it's more idiomatic golang to do it that way.

Reply:
ok

Reply:
fixed

Reply:
Thanks. Thank you for your contribution and thank you for your patience with our requests.

Reply:
My pleasure. I want the code to be perfect. Thanks.

Reply:
All CI's passed